history/history-api/src/main/java/com/duckduckgo/history/api/NavigationHistory.kt

```diff
@@ -39,4 +39,19 @@ interface NavigationHistory {
      * Clears all history entries.
      */
     suspend fun clearHistory()
+
+    /**
+     * Returns whether the history is enabled by the user.
+     */
+    fun isHistoryUserEnabled(): Boolean
+
+    /**
+     * Sets whether the history is enabled by the user.
+     */
+    fun setHistoryUserEnabled(value: Boolean)
+
+    /**
+     * Returns whether the history RC flag is enabled.
+     */
+    fun isHistoryRCFlagEnabled(): Boolean
```

Comment:
What about naming it as `isHistoryUserVisible`? just nit to name it based on what we expect rather than what we internally check.

Reply:
I'd rather change it to `isHistoryEnabled`, `userVisible` sounds too close to the UI layer for an API method